document custom identity providers (#11)
integr8ly/user-documentation: docs/content/gs-custom-keycloak-idp.adoc

[id='gs-custom-keycloak-idp']

ifdef::env-github[]
:imagesdir: ../images/
endif::[]

> = Adding custom identity providers to Integreatly SSO

Reviewer: ```suggestion
= Adding custom identity providers to {PRODUCT}
```

> Integreatly comes preinstalled with Single Sign on based on link:https://www.keycloak.org/[Keycloak^].

Reviewer: ```suggestion
By default, {PRODUCT} includes a Single Sign-On solution using Red Hat Single Sign-On.
```

This lets you log in to various middleware services using the same accounts.

> Keycloak allows you to supply your own login mechanism using Identity Providers. This document describes how to add a custom Identity Provider to the Integreatly Single Sign On.

Reviewer: ```suggestion
RHSSO allows you to supply your own login mechanism using Identity Providers. This section describes how to add a custom Identity Provider to the Integreatly Single Sign-On.
```

> This guide assumes that you are logged in to OpenShift with the `customer-admin` user.

Reviewer: ```suggestion
.Prerequisite
You are logged in to OpenShift as the `customer-admin` user.
```
